zeppelin-interpreter-parent/pom.xml

```diff
@@ -36,6 +36,7 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>zeppelin-interpreter-shaded</artifactId>
       <version>${project.version}</version>
+      <scope>provided</scope>
```

Comment:
this looks correct to me, as `zeppelin-interpreter-shaded-*.jar` will always be added to the interpreter process's classpath.

https://github.com/apache/zeppelin/blob/940cf13935ec9ed989c7afeec98447c2e7eb5a7e/bin/interpreter.sh#L109-L110

Reply:
with this change, some interperters pom.xml `<exclude>org.apache.zeppelin:zeppelin-interpreter-shaded</exclude>` can be removed

Reply:
Possibly, this should be investigated in more detail. @jongyoul  once suggested that zeppelin-interpreter and zeppelin-interpreter-shaded should be merged, if I remember correctly